utils/release.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+VER=$(cat ./VERSION)
+echo "- Current version: ${VER}"
+echo -n "New version (e.g 50.0): "
+read -r NEW_VER
+
+echo "- Updating ./VERSION"
+echo -n "$NEW_VER" >./VERSION
+
+echo "- Updating ./meson.build"
+sed -i -e "s/version: '${VER}'/version: '${NEW_VER}'/g" ./meson.build
+
+FILE=./data/com.usebottles.bottles.metainfo.xml.in
+POS=$(grep -n "<releases>" ${FILE} | cut -f1 -d:)
+
+echo "- Adding stub release description to ${FILE}"
+sed -i -e "${POS}r /dev/stdin" ${FILE} <<EOT
+        <release version="${NEW_VER}" date="$(date -Idate)">
```

Comment:
don't forget to set the translateable bit here, so translators don't waste time translating descriptions. https://github.com/bottlesdevs/Bottles/pull/3113